lighthouse-core/report/html/renderer/base64.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* global self btoa atob pako CompressionStream Response */
+
+const encode = typeof btoa !== 'undefined' ?
+  btoa :
+  /** @param {string} str */
+  (str) => Buffer.from(str).toString('base64');
+const decode = typeof btoa !== 'undefined' ?
+  atob :
+  /** @param {string} str */
+  (str) => Buffer.from(str, 'base64').toString();
+
+function getPako() {
+  return typeof pako !== 'undefined' ? pako : require('pako');
```

Comment:
what sets up this global? can we get a little explanation in the jsdoc here about where this is coming from?

also I don't think we've ever included a `require` statement in the `report/html` JS before this seems like a recipe for confusion on whether this is bundled with node_modules or not (I don't think it ever is?). Node tests might be a reason to do this, but even then we can setup the `pako` global like we do for all other renderer JS tests.

Reply:
hahah yeahh it was here for node tests, but now the tests skip gzip b/c we decided to go wit CompressionStream (Which is not in node), so i'll remove this.